### PR TITLE
'istioctl proxy-config clusters' output regression

### PR DESCRIPTION
**Pull request by esnible**
Resolves https://github.com/istio/istio/issues/12455

### CODE REVIEW

File: istioctl/pkg/writer/envoy/configdump/cluster.go [hunk 1]
```diff
@@ -73,9 +73,9 @@ func (c *ConfigWriter) PrintClusterSummary(filter ClusterFilter) error {
 				if subset == "" {
 					subset = "-"
 				}
-				fmt.Fprintf(w, "%v\t%v\t%v\t%v\t%v\n", fqdn, port, subset, direction, cluster.ClusterDiscoveryType)
+				fmt.Fprintf(w, "%v\t%v\t%v\t%v\t%s\n", fqdn, port, subset, direction, cluster.GetType())
```
Comment (rshriram) [inline istioctl/pkg/writer/envoy/configdump/cluster.go]: @kyessenov made a recent change where this was changed from getType to clusterDiscoveryType. You should consult with him..

**Comment by esnible** [comment]
@kyessenov Requesting your review because I see no reason to change the documented output to wrap the type visually in `&{...}`.

It is possible this PR is insufficient because it doesn't address custom Envoy cluster types.  I haven't been able to locate an example of that.

**Comment by esnible** [comment]
I need someone at Google to remove the `cla: no` tag.  I have noticed that if I create a PR and then change the base from `master` to `release 1.1` within the first few seconds I falsely get this tag.

**Comment by esnible** [comment]
@kyessenov Your LGTM didn't make it through.  Can you remove the bogus `CLA: no` (only Googlers can do this).  Can you also give me a positive review as you have approved it?
@linsun Who can get this merged?

**Comment by kyessenov** [comment]
1.1 is admin protected, so my commands won't work.

**Comment by esnible** [comment]
@kyessenov @geeknoid Any chance of a positive review?

**Comment by kyessenov** [comment]
/lgtm
/approve
... but my powers don't extend to istioctl...

**Pull request merged**